docs/domains/Report/services/ReportService/openapi.yml

```
@@ -53,6 +53,56 @@
+          type: string
+      responses:
+        "200":
+          description: OK
+        "400":
+          description: Bad Request
+        "500":
+          description: Internal Server Error
+  /api/Report/Submission:
+    get:
+      tags:
+      - Report
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 8 but found 6

(indentation)